examples/hello-world/blueprint.yaml

```diff
@@ -0,0 +1,64 @@
+# The information below is boilerplate for the blueprint.
+apiVersion: xl/v1
+kind: Blueprint
+
+# This metadata helps the next user better understand what the blueprint is about.
+
+metadata:
+  projectName: HelloWorld
+  description: |
+    The blueprint creates a Hello World application, environment, and infrastructure for the purpose of illustrating how a 
+    blueprint works.  We also use XL Release to orchestrate the distribution of this software to environments, so you can 
+    also see how to plumb-in XLR into a blueprint.
+  author: XebiaLabs
+  version: 1.0
+  
+parameters:
+# The blueprint mechanism will prompt the user for these values.  The specification for these parameters start at the following location:
+# https://docs.xebialabs.com/xl-platform/concept/blueprint-yaml-format.html
+# See also the real-world example of how some of these are organized at:
+# https://github.com/xebialabs/blueprints/blob/master/aws/microservice-ecommerce/blueprint.yaml
+# The values below are going to substituted into different XLD and XLR places.  The questions are meant to help explain how the value will be used?
+# XLD substitutions
+
+- name: appName
+  description: What is the name of the Application?
+  type: Input
+  default: HelloWorldApp
+- name: envName
```

Comment:
I think you can derive an env name and infra name based on app name. This is against DoD